plugins/inputs/win_eventlog/README.md

```diff
@@ -11,6 +11,17 @@ Windows Events Channels, like System Log.
 
 Telegraf minimum version: Telegraf 1.16.0
 
+## Service Input <!-- @/docs/includes/service_input.md -->
```

Comment:
I think this one is not a service plugin.

Reply:
done